boot.php

```diff
@@ -15,9 +20,9 @@
     throw new RuntimeException('You need to pass the ROM file name (Ex: drmario.rom)');
 }
 
-$filename = 'roms/'.$argv[1];
+$filename = getcwd().'/'.$argv[1];
```

Comment:
Symfony/console is for encapsulate the command and the emulator, providing a more eficient way to interact with emulator, add more commands, etc